com.microsoft.mrtk.input/InteractionModes/FlatScreenModeDetector.cs

```diff
@@ -22,7 +22,7 @@ internal class FlatScreenModeDetector : MonoBehaviour, IInteractionModeDetector
 
         public void Awake()
         {
-            ControllerLookup[] lookups = GameObject.FindObjectsOfType(typeof(ControllerLookup)) as ControllerLookup[];
+            ControllerLookup[] lookups = ComponentCache<ControllerLookup>.FindInstanceList();
```

Comment:
Can we change this to be FindFirstInstance() instead of the list.   Allocating an array just to provide a warning seems like a waste of resources.